apps/saleor-app-checkout/test-utils.ts

```diff
@@ -11,6 +11,7 @@ import { MockedRequest } from "msw";
 declare module "next" {
   type NextApiResponse = {
     _getJSONData: <T extends Object>() => T;
+    _getData: <T extends string>() => T;
   } & NextApiResponse;
 }
```

Comment:
We shouldn't be doing this. It affects the whole codebase, not just tests. Let's remove this type and instead assert:

```suggestion
export type TestNextApiResponse = NextApiResponse & {
  _getJSONData: <T extends Object>() => T;
  _getData: <T extends string>() => T;
};
```

Reply:
Done, I've changed it